src/consensus/pbft/libbyz/test/replica_main.cpp

```diff
@@ -202,6 +203,14 @@ int main(int argc, char** argv)
     LOG_FATAL << "--transport {UDP || UDP_MT}" << std::endl;
   }
 
+  auto store = std::make_shared<ccf::Store>(
+    pbft::replicate_type_pbft, pbft::replicated_tables_pbft);
+  auto& pbft_requests = store->create<pbft::Requests>(
+    pbft::Tables::PBFT_REQUESTS, kv::SecurityDomain::PUBLIC);
+  auto& pbft_pre_prepares = store->create<pbft::PrePrepares>(
+    pbft::Tables::PBFT_PRE_PREPARES, kv::SecurityDomain::PUBLIC);
+  auto replica_store = std::make_unique<pbft::Adaptor<ccf::Store>>(store);
+
```

Comment:
Are we running any form of recovery for these tests? I think this can just be stubbed out.

Reply:
No but we need to pass references to tables so it can't be nullptr here anymore. What do you mean stubbed out?

Reply:
have an abstract base class or adapter class that wraps the functionality that you need from the Table class. It would be good to reduce coupling if we can.